services/masterdata/src/lib/core/adapters/translation/languages-result-list-model.adapter.ts

```diff
@@ -0,0 +1,21 @@
+import { IAdapter, ISourceLanguagesResultList } from "../../interfaces";
+import { LanguageItem, LanguagesResultList } from "../../models";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export class LanguagesResultListModelAdapter implements IAdapter<any, LanguagesResultList> {
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
eslint comment can be removed + any on line 5 can be replaced with the ISourceLanguagesResultList